subprojects/docs/src/docs/userguide/upgrading_version_5.adoc

```diff
@@ -73,6 +73,12 @@ java {
 }
 ```
 
+==== Bugfix in Maven / Ivy interoperability with dependency substitution
+
+There was a bug on how metadata was processed when an Ivy dependency was substituted (using a `resolutionStrategy.force`, `resolutionStrategy.eachDependency` or `resolutionStrategy.dependencySubstitution`)
+and was referenced by a Maven dependency.
+This would still trigger the pre-Gradle 5.0 resolution instead of searching for the Ivy `default` configuration.
```

Comment:
:x: This should be rephrased, it doesn't really ring a bell to me. What does this mean?

Reply:
I am taking ideas. While it is a potential breaking change, it has a very narrow scope: using Maven to Ivy dependency, using substitution on the Ivy dependency, having an Ivy dependency where `default` dependencies are different than the sum of `compile`, `runtime` and `master`.
Should I use the above? Or is there a more compact way of expressing it that will draw attention from users in that situation?

Reply:
Yeah I would describe exactly that. Or basically I don't know if I'm affected or not.

Reply:
I have updated the description, please take another look.